docs/guide/quick-start.txt

```diff
@@ -0,0 +1,327 @@
+===========
+Quick Start
+===========
+This guide will show you how to set up a simple application using Node.js and MongoDB. Its scope is only how to set up the driver and perform the simple CRUD operations.
```

Comment:
There used to be a link saying ` For more in-depth coverage, see the tutorials.`, with a link to http://mongodb.github.io/node-mongodb-native/3.3/reference/main/. Since we are eliminating that page, should we have an equivalent link to something else here?

Reply:
I'd like to have as much cross-linking as possible. Can you just leave yourself a placeholder reminder for now? Perhaps one that doesn't render?

Reply:
Added https://jira.mongodb.org/browse/NODE-2244 to follow up.